libs/config/deno_json/mod.rs

```diff
@@ -1850,6 +1856,48 @@ impl ConfigFile {
       }
     }
   }
+
+  pub fn dependencies(&self) -> HashSet<JsrDepPackageReq> {
```

Comment:
Moved from resolver/lockfile.rs

Reply:
I still find it confusing that `dependencies` returns `JsrDepPackageReq` even though it's a superset that contain npm packages too